MovingBlocks/Terasology
# Renderer work on camera-related StateChanges

Adds and takes advantage of a number of StateChanges and associated tasks:
- LookThrough
- LookTroughNormalized
- SetCameraToReflected

### How to test
- run Terasology
- create a new game or load an existing one
- verify nothing has changed in the graphic output
